cmd/ctr/checkpoint.go

```diff
@@ -67,6 +68,24 @@ var checkpointCommand = cli.Command{
 		if err := json.Unmarshal(info.Spec.Value, &spec); err != nil {
 			return err
 		}
+		stopped := context.Bool("exit")
+		// if the container will still be running after the checkpoint make sure that
+		// we pause the container and give us time to checkpoint the filesystem before
+		// it resumes execution
+		if !stopped {
+			if _, err := containers.Pause(ctx, &execution.PauseRequest{
+				ID: id,
+			}); err != nil {
+				return err
+			}
+			defer func() {
+				if _, err := containers.Resume(ctx, &execution.ResumeRequest{
```

Comment:
I think we don't need to Resume() a container if it was paused by user before calling Checkpoint().